modules/web/server/src/main/scala/observe/web/server/config/package.scala

```diff
@@ -36,49 +34,48 @@ package config {
  */
 package object config {
 
-  implicit val siteReader = ConfigReader.fromCursor[Site] { cf =>
+  implicit val siteReader: AnyRef with ConfigReader[Site] = ConfigReader.fromCursor[Site] { cf =>
```

Comment:
Why the `AnyRef`?

Reply:
I don't remember writing that. I think it was put when I used an automatic fix for untyped public vars